spec/fabricators/user_fabricator.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'ffaker'
```

Comment:
I think we don't need to require it, it should be available automatically here

Reply:
When I read comments like this, I feel like "beginner mistake" ! :P 
Thanks!